composer/spec/dependabot/composer/file_fetcher_spec.rb

```diff
@@ -76,6 +83,12 @@
     it "fetches the composer.json" do
       expect(file_fetcher_instance.files.map(&:name)).to eq(["composer.json"])
     end
+
+    it "provides the composer version" do
+      expect(file_fetcher_instance.package_manager_version).to eq({
+        ecosystem: "composer",
+        package_managers: { "composer" => "2" }
```

Comment:
Vs this test will doublecheck that if we don't have `composer.lock` so we can't know the full version and are defaulting to `2` that it still continues to default even if we return the full version normally.